src/wallet/wallet.ts

```diff
@@ -527,9 +704,8 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
    * @memberof HathorWalletServiceWallet
    * @inner
    */
-  getInputData(dataToSignHash: Buffer, addressPath: string): Buffer {
-    const code = new Mnemonic(this.seed);
-    const xpriv = code.toHDPrivateKey(this.passphrase, this.network.bitcoreNetwork);
+  getInputData(xprivkey: string, dataToSignHash: Buffer, addressPath: number): Buffer {
```

Comment:
Since we don't store the seed on memory anymore, we need to receive a xprivkey to sign the inputs.

This `xpriv` is stored on LocalStorage when the wallet facade is initialized (on the `start` method) using the `initializeAccessData` method.